template.php

```diff
@@ -68,6 +76,11 @@ function ochabasic_preprocess_html(&$vars) {
       'color' => '#5bbad5',
     ),
   );
+
+  drupal_add_html_head($viewport, 'viewport');
+  drupal_add_html_head($apple, 'apple-touch-icon');
+  drupal_add_html_head($fav_32, 'favicon-32x32');
+  drupal_add_html_head($fav_16, 'favicon-16x16');
   drupal_add_html_head($safari_pinned_tab, 'safari_pinned_tab');
 }
 
```

Comment:
@rupl I had a merge conflict here. I think I corrected it as I should, based on master. Please double check